### ISSUE TITLE
Sublime Text 3

### ISSUE THREAD

**Issue by jhpaul** [SubZane/Sublime-Placehold.it#5]
manual install didn't work in ST3 and it's not yet in package control. Any chance we could see an update?

Let me know if I can help. I poked around the code, but I don't know enough at the moment to fix it.

Thanks

**Comment by tiagoporto** [comment]
+1

**Comment by SubZane** [comment]
I've moved over to Atom.io and I'm not very likely to fix this anytime soon. Is  anybody up for the task to take over this project/repo in order for this plugin to live?

**Comment by tiagoporto** [comment]
Hello @SubZane, I started to fix, but I'm not very good with python, if someone can help will be great.

**Comment by bartholomej** [comment]
Hi guys, we've just pushed new update with support for ST3 (not yet available via official Sublime Package Repository). Can anyone try to install manually and confirm? ;)

**Comment by tiagoporto** [comment]
Hello, I tested it in sublime build 3083 Win 8.1.

With the command `p + h + tab` works, but with delay.

In the the `command palette` doesn't display any command.

And in menu `preferences> package settings > placehold.it > settings`  the file opens blank.

**Comment by bartholomej** [comment]
@tiagoporto Thanks for your feedback.
Please try new update and make sure the directory path does not contain _dot_.
This is correct directory name:  `Packages/Placeholdit Image Tag Generator/`

**Comment by Jakobud** [comment]
@SubZane could transfer ownership of this repo to @bartholomej if @bartholomej wants to take it over. @bartholomej will need to delete or rename his fork first though.

**Comment by bartholomej** [comment]
@Jakobud Yes I know that. But if @SubZane wants to transfer ownership we prefer to transfer it to our _team account_: @AquaSoup. We'll  take care of it.

**Comment by tiagoporto** [comment]
@bartholomej Much better, but I checked a frustrating thing. Every inserted image, it's created a directory with the same filename opened, and inside other directory images with the image files, example index.html/images/placehold-600x400-0eafff-ffffff.png.

**Comment by Jakobud** [comment]
Where is this forked version and how do I install it? I'd like to test it out to give feedback.

**Comment by pafnuty** [comment]
@Jakobud 
@tiagoporto 
try [this plugin](https://packagecontrol.io/packages/imgHolder)

**Comment by Jakobud** [comment]
I'm talking about the placehold.it plugin fork that @bartholomej is working on maintaining.

**Comment by tiagoporto** [comment]
Thanks @pafnuty.

**Comment by bartholomej** [comment]
@tiagoporto You're right! Should be fixed now. It was a problem with path on Windows. And _delay_ problem should be fixed also. Can you confirm please?
@Jakobud There is no forked repository anymore. Everything is in this repository _dev branch_. Master is up to date now. Let's try ;)

Thanks guys

**Comment by Jakobud** [comment]
How do I install this dev branch without Package Control?

**Comment by bartholomej** [comment]
@Jakobud and anyone who wants to install package **without Package Control**:
1. Use `Preferences` > `Browse Packages...` to open _packages folder_.
2. Take your Plugin (which you can download as a zip from this github repository) and simply copy the folder into your _Packages folder_. (Or just clone this repository into this folder).
3. Make sure the name of this folder is `Packages/Placeholdit Image Tag Generator/`
4. Have fun and give a feedback ;)

**Comment by Jakobud** [comment]
Seems to work in ST3. Not broken that I can see. Some functionality feedback/suggestions:
- To do anything you have to Command Palette > type in placehold.it, and then it gives you 3 options for inserting default, custom or cached images. I would recommend getting rid of the first step and just having 3 different commands you can access directly in the Command Palette: Placehold.it: Insert Default Image, Placehold.it Insert Custom Size Image, Placehold.it Insert Cached Image. This is similar to how package control commands are setup. You don't have to type "Package Control" and then you are selected with another list of commands. It gives you all the commands immediately. I would recommend a similar approach.
- I would maybe get rid of the Insert Default Image command... I mean, I don't really see the reason for having that. When someone is putting together a webpage its unlikely they just happen to need the exact same dimension image as the default and would use this. I would probably never use it personally. I think that Custom should be the default goto approach.
- When choosing Custom Size, it gives me 4 sizes of images to choose from, but I can't figure out how to actually choose a custom size that isn't in that list.... am I missing something? I type in something like 150x800 but it doesn't do anything.
- It would be nice if there was a place to specify custom text, custom text color and custom background color.

**Comment by Jakobud** [comment]
Also why do you have to have a saved file in order to use the plugin? You can't just create a blank document and insert a placehold.it image tag. What is the reason for this restriction?

**Comment by tiagoporto** [comment]
Hello guys. @bartholomej  the problem with delay has been resolved, but the directory is still created.

**Comment by bartholomej** [comment]
Hi @tiagoporto yes because default value in settings was set on true. Please try it again (dev branch).
I think everything works fine. Ready for production? :)

**Comment by tiagoporto** [comment]
Hello @bartholomej, sorry for the delay, I am using [imgHolder](https://packagecontrol.io/packages/imgHolder) now. But I tested the dev branch and did not work anything.

**Comment by bartholomej** [comment]
Hi @tiagoporto are you sure the name of this folder is exactly `Packages/Placeholdit Image Tag Generator/`?
I'm using this plugin (dev branch) all the time and it works well for me. Thx
